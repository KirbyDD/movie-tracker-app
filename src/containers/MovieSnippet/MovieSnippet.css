.snippet {
  margin: 10px;
}

.poster {
  width: 200px;
  box-shadow: 0 0 10px 2px rgba(0, 0, 0, 0.068);
}

.poster:hover {
  box-shadow: 0 0 10px 5px rgba(0, 0, 0, 0.137);
  border: 1px solid #fff;
}