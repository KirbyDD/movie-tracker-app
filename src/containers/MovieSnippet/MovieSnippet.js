import React from 'react';
import { Link } from 'react-router-dom';
// import { connect } from 'react-redux';
import './MovieSnippet.css'

const MovieSnippet = (props) => {
  const url = 'https://image.tmdb.org/t/p/w500'
  return (
    <Link to={`/movie/${props.id}`} className='snippet'>
      <img src={url+props.path} className='poster' alt="movie snippet"/>
    </Link>
  )
}

export default MovieSnippet;