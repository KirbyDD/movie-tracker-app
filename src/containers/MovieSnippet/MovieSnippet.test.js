import React from 'react';
import MovieSnippet from './MovieSnippet';
import {shallow} from 'enzyme';

describe('Movie Snippet', () => {
  let wrapper;
  let props;

  beforeEach(function() {
    props = { id: 1, path: '/w9kR8qbmQ01HwnvK4alvnQ2ca0L.jpg'}
    wrapper = shallow(
      <MovieSnippet id={props.id} path={props.path} />
    )
  })
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})