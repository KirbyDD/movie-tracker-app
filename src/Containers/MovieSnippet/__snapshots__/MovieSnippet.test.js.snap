// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Snippet should match snapshot 1`] = `
<Link
  className="snippet"
  to="/movie/1"
>
  <img
    alt="movie snippet"
    className="poster"
    src="https://image.tmdb.org/t/p/w500/w9kR8qbmQ01HwnvK4alvnQ2ca0L.jpg"
  />
</Link>
`;
